lib/resourceapply/machineconfig_test.go

```
@@ -296,3 +298,93 @@ func TestApplyMachineConfig(t *testing.T) {
+		Spec: mcfgv1.MachineConfigNodeSpec{
+			Pool: mcfgv1.MCOObjectReference{Name: "master"},
+		},
+	}
+
+	client := fake.NewSimpleClientset(existing)
+	conflictErr := apierrors.NewConflict(
+		schema.GroupResource{Group: "machineconfiguration.openshift.io", Resource: "machineconfignodes"},
+		"node-1",
+		fmt.Errorf("the object has been modified; please apply your changes to the latest version and try again"),
+	)
+	client.PrependReactor("update", "machineconfignodes", func(action clienttesting.Action) (bool, runtime.Object, error) {
```

golangci-lint (2.13.2)

[warning] unused-parameter: parameter 'action' seems to be unused, consider removing or renaming it as _

(revive)